src/JitInterface/src/CorInfoImpl.Intrinsics.cs

```diff
@@ -190,6 +192,12 @@ private CorInfoIntrinsics getIntrinsicID(CORINFO_METHOD_STRUCT_* ftn)
                     }
                     break;
 
+                // TODO: Implementation of pMustExpand in RyuJIT is not correct
```

Comment:
IsIntrinsicImplementedByUserCall is true for these two intrinsics that causes RyuJIT to fail fast for them, even though everything is fine.